core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -70,6 +70,14 @@ THE SOFTWARE.
                     </j:forEach>
                   </td>
                   <j:if test="${!x.folder}">
+                    <td class="fileSize">
+                      <j:if test="${x.lastModifiedAsCalendar.timeInMillis &gt; 0}">
+                        <i:formatDate value="${x.lastModifiedAsCalendar.time}" type="both" dateStyle="medium" timeStyle="medium"/>
+                      </j:if>
+                      <j:if test="${x.lastModifiedAsCalendar.timeInMillis &lt;= 0}">
+                        ${%No available}
```

Comment:
*N/A* was OK and consistent with other UI. *No available* OTOH is ungrammatical.